src/deploy/NVA_build/builder.Dockerfile

```diff
@@ -18,8 +18,8 @@ RUN version="1.3.0" && \
     tar -xf yasm-${version}.tar.gz && \
     pushd yasm-${version} && \
     ./autogen.sh && \
-    make && \
-    make install && \
+    make -j`nproc` && \
+    make -j`nproc` install && \
```

Comment:
Liran tested and this yasm make -j can shave off 12 seconds.
Unless this is critical I would avoid make -j for makefiles that are not in our control as it is fragile.